extensions/linear/src/components/ProjectIssues.tsx

```diff
@@ -18,9 +22,54 @@ type ProjectIssuesProps = {
 
 export default function ProjectIssues({ projectId, priorities, me, users }: ProjectIssuesProps) {
   const { issues, isLoadingIssues, mutateList } = useIssues(getProjectIssues, [projectId]);
+  const [milestone, setMilestone] = useCachedState<string>("");
+  const { milestones } = useMilestones(projectId);
+
+  const filteredIssues = useMemo(() => {
+    if (!issues) {
+      return [];
+    }
+
+    if (milestone === "") {
+      return issues;
+    }
+
+    const filteredIssues = [];
+    for (const issue of issues) {
+      if (issue.projectMilestone && issue.projectMilestone?.id === milestone) {
+        filteredIssues.push(issue);
+      }
+    }
+
+    return filteredIssues || [];
+  }, [milestone, issues, milestones]);
 
   return (
-    <List isLoading={isLoadingIssues} searchBarPlaceholder="Filter by ID, title, status, assignee or priority">
+    <List
+      isLoading={isLoadingIssues}
+      {...(milestones && milestones.length > 0
+        ? {
+            searchBarAccessory: (
+              <List.Dropdown tooltip="Change Roadmap" onChange={setMilestone} value={milestone}>
```

Comment:
Shouldn't it be `Change Milestone` instead?

```suggestion
              <List.Dropdown tooltip="Change Milestone" onChange={setMilestone} value={milestone}>
```

Reply:
Yes! Good catch!